inc/vcf/error.hpp

```diff
@@ -36,14 +55,9 @@ namespace ebi
                   message{message} {}
 
 
-        size_t get_line() const
-        {
-          return line;
-        }
-        const std::string &get_raw_message() const
-        {
-          return message;
-        }
+        size_t get_line() const { return line; }
+        const std::string &get_raw_message() const { return message; }
+        virtual ErrorCode get_code() const { return ErrorCode::error; }
```

Comment:
yes, I didn't think about it, it makes the code a bit more elegant. fixed in next commit